frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -30,12 +30,15 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
             IrTypeDelegate::PrimitiveEnum { ref repr, .. } => {
                 format!("return _api2wire_{}(raw.index);", repr.safe_ident())
             }
+            IrTypeDelegate::Backtrace => unimplemented!(),
```

Comment:
TODO(fzyzcjy): Check whether valgrind suppress is ok

(unrelated to this line of code, just a reminder for me)